packages/panzer/disc-fe/test/evaluator_tests/normals.cpp

```diff
@@ -228,7 +228,8 @@ typedef Traits::Residual ResidualType;
 typedef Traits::Jacobian JacobianType;
 
 UNIT_TEST_GROUP(ResidualType)
-UNIT_TEST_GROUP(JacobianType)
+// Commented out to avoid ICE in Intel 2021.3.0
+//UNIT_TEST_GROUP(JacobianType)
```

Comment:
It is surprising that these tests need to be disabled for this Intel 19.0.5 build.  Are there not other nightly Intel 19 builds running already that run these Panzer tests?

Reply:
Again, this was for Intel 2021.3.0. It's fine on 19.0.5

Reply:
That failure is tracked here: https://github.com/trilinos/Trilinos/issues/10097